src/data/ACTIONS/root/RDM.ts

```diff
@@ -69,7 +69,33 @@ export const RDM = ensureActions({
 		name: 'Fleche',
 		icon: iconUrl(3212),
 		cooldown: 25000,
-		potency: 460,
+		potency: 480,
+	},
+	VICE_OF_THORNS: {
+		id: 37005,
+		name: 'Vice of Thorns',
+		icon: iconUrl(3242),
+		potency: 700,
+	},
+	GRAND_IMPACT: {
+		id: 37006,
+		name: 'Grand Impact',
+		icon: iconUrl(3243),
+		potency: 600, // Gives 3 white, 3 black
+	},
+	PREFULGENCE: {
+		id: 37007,
+		name: 'Prefulgence',
+		icon: iconUrl(3244),
+		potency: 900,
+	},
+	MAGICK_BARRIER: {
+		id: 25857,
+		name: 'Magick Barrier',
+		icon: iconUrl(3237),
+		speedAttribute: Attribute.SPELL_SPEED,
```

Comment:
I'm fairly sure this cooldown doesn't scale with speed, it's just the instant OGCD defensive, correct?

I guess you moved an existing value, but probably worth correcting it while we're here.

Reply:
I'll remove, I didn't even look at it; I just noticed it was down with the casted GCDs instead of with all the ability oGCDs.